docs/powergate/storage.md

```diff
@@ -0,0 +1,158 @@
+# Storing Data
+
+Within the Powergate API, Storage Profiles manage all the necessary state and capabilities to provide multi-tiered file storage. The primary API for storing and retrieving data, tracking long-term deals on Filecoin, and allowing data persisted on Filecoin to be available on IPFS is managed through and scoped to a Storage Profile. 
+
+## Intro to Storage Profiles
+
+Storage profiles are scoped to one or more Filecoin wallet addresses. So to start using a Storage Profile, you must use the Powergate admin API to create a new Storage Profile, at which time the Powergate will:
+
+1. Create a new default wallet address for the Storage Profile. You can configure the Powergate to automatically fund new wallets from a master address.
+2. Create a new API token linked to the Storage Profile.
+3. Enable use of the Storage Profile through the use of the supplied token.
+
+Almost all Powergate APIs rely on the Storage Profile (including use through the CLI), so you will need to supply the _token_ to indicate which _Storage Profile_ your requests are targeting. Since each Storage Profile has its own address, it has its own balance and therefore limits on the Filecoin network.
+
+!!!Warning
+    If you're providing a `--lotusmasteraddr` and `--walletinitialfund`, be sure that address exists in the Lotus node and it has enough funds, since `walletinitialfund` attoFILs will be sent from there to fund from newly created Storage Profiles. Recall that both flags are optional, and if not present there won't be any auto-funding transaction, so you're responsible to fund wallet addresses of new Storage Profiles. 
+
+## Multi-tiered design
+
+Powergate provides you API access to multi-tiered storage system built on IPFS and Filecoin. In many places, we refer to these two tiers of storage as Hot (IFPS) and Cold (Filecoin). This mirrors multi-tiered storage often deployed with a hot storage layer in _memory_ and a cold storage layer on _disk_.
+
+### Hot storage layer
+
+Data stored in the Powergate hot layer is available to the IPFS network (or private network). Hot storage is designed to be fast and available on the IPFS network (private or public DHT). The default `StorageConfig` enables both hot and cold for all new data stored. Data stored with hot enabled is pinned to the Powergate's IPFS node. 
```

Comment:
> The default `StorageConfig` enables both hot and cold for all new data stored

This isn't true now that we have _HotStorage_ disabled by default. Maybe it's better to remove this sentence to not tie the docs with some changing decisions we might make?